node/node.go

```diff
@@ -414,7 +414,8 @@ func (nc *Config) Build(ctx context.Context) (*Node, error) {
 	}
 
 	// Set up libp2p network
-	fsub, err := libp2pps.NewFloodSub(ctx, peerHost)
+	// TODO: we probably want to signe messages
+	fsub, err := libp2pps.NewFloodSub(ctx, peerHost, libp2pps.WithMessageSigning(false))
```

Comment:
Issue for this here: https://github.com/filecoin-project/go-filecoin/issues/3124 will add to comment in code as well